enos/enos-modules.hcl

```diff
@@ -26,6 +26,18 @@ module "boundary" {
   alb_listener_api_port = var.alb_listener_api_port
 }
 
+module "solo_worker" {
```

Comment:
Do we need to add `project_name` here? I see it in the `boundary` module and it's defined in `variables.tf`, but I don't see it used anywhere?

Reply:
I've added this to be piped in to the module, but I don't really have a use for it. Still, it might be useful to have as this module gets used, but it doesn't serve a purpose right now other than mirroring the setup it came from.